frontend/src/Components/LegalNotice/LegalNotice.tsx

```diff
@@ -2,116 +2,190 @@ import classes from './LegalNotice.module.css';
-
-    <h2>Widerruf Ihrer Einwilligung zur Datenverarbeitung</h2>
+    <h2>3. Data Processing</h2>
     <p>
-      Nur mit Ihrer ausdrücklichen Einwilligung sind einige Vorgänge der Datenverarbeitung möglich.
-      Ein Widerruf Ihrer bereits erteilten Einwilligung ist jederzeit möglich. Für den Widerruf
-      genügt eine formlose Mitteilung per E-Mail. Die Rechtmäßigkeit der bis zum Widerruf erfolgten
-      Datenverarbeitung bleibt vom Widerruf unberührt.
+      Here we inform you about how we process your personal data. "Personal data" in this context
+      includes all information relating to an identified or identifiable natural person.
+      "Processing" means all operations or series of operations relating to personal data, such as
+      the collection or storage of personal data.
     </p>
-
-    <h2>Recht auf Beschwerde bei der zuständigen Aufsichtsbehörde</h2>
+    <h3>3.1 Website hosting</h3>
     <p>
-      Als Betroffener steht Ihnen im Falle eines datenschutzrechtlichen Verstoßes ein
-      Beschwerderecht bei der zuständigen Aufsichtsbehörde zu. Zuständige Aufsichtsbehörde bezüglich
-      datenschutzrechtlicher Fragen ist das Bayerisches Landesamt für Datenschutzaufsicht:{' '}
-      <a href="https://www.lda.bayern.de/de/kontakt.html" target="_blank">
-        https://www.lda.bayern.de/de/kontakt.html
-      </a>
-      .
-    </p>
-
-    <h2>Recht auf Datenübertragbarkeit</h2>
-    <p>
-      Ihnen steht das Recht zu, Daten, die wir auf Grundlage Ihrer Einwilligung oder in Erfüllung
-      eines Vertrags automatisiert verarbeiten, an sich oder an Dritte aushändigen zu lassen. Die
-      Bereitstellung erfolgt in einem maschinenlesbaren Format. Sofern Sie die direkte Übertragung
-      der Daten an einen anderen Verantwortlichen verlangen, erfolgt dies nur, soweit es technisch
-      machbar ist.
+      Each time you visit our website, we automatically collect and store the following data using
+      Amazon Web Services, Inc. (AWS):
+      <ul>
+        <li>IP address</li>
+        <li>Date and time of access</li>
+        <li>Information about your browser and operating system</li>
+      </ul>
+      This data will be automatically deleted after 7 days. Exceptions to this rule will only be
+      made in cases where there is a legitimate interest in its further storage, for example to
+      clarify security incidents. In these cases, the data will be deleted as soon as this process
+      is completed.
+      <br />
+      <br />
+      This data processing is based on our legitimate interest to make our website accessible to
+      users, to identify any functional problems or capacity deficits of our website and to verify
+      security in accordance with Art. 6 para. 1 lit. f DS-GVO/UK GDPR.
     </p>
-
-    <h2>Recht auf Auskunft, Berichtigung, Sperrung, Löschung</h2>
+    <h3>3.2 Login and Voting Results</h3>
+    The (freely chosen) Login Name and Your Voting Results are used to distribute them to the voting
```

Comment:
```suggestion
    The chosen login name and votes are being distributed to other attendees of the same voting session.
```

Reply:
since it is approved with data protection I would not change it too much and it gives the user the hind that the login name can be arbitrary